internal/cmd/api-key/command_store.go

```diff
@@ -36,21 +36,13 @@ func (c *command) newStoreCommand() *cobra.Command {
 				Text: "Pass the API key and secret as arguments",
 				Code: "confluent api-key store my-key my-secret",
 			},
-			examples.Example{
-				Text: "Get prompted for both the API key and secret",
-				Code: "confluent api-key store",
-			},
 			examples.Example{
 				Text: "Get prompted for only the API secret",
 				Code: "confluent api-key store my-key",
 			},
 			examples.Example{
-				Text: "Pipe the API secret",
-				Code: "confluent api-key store my-key -",
-			},
-			examples.Example{
-				Text: "Provide the API secret in a file",
-				Code: "confluent api-key store my-key @my-secret.txt",
+				Text: "Get prompted for both the API key and secret",
+				Code: "confluent api-key store",
 			},
```

Comment:
This style of input differs from the rest of our commands, we should not recommend this. Users can use `cat my-secret.txt | confluent api-key store my-key` instead.